### PR TITLE
Documentation: Document behaviour for existing pods

### PR DESCRIPTION
**Pull request by tgraf**
Signed-off-by: Thomas Graf <thomas@cilium.io>

### CODE REVIEW

File: Documentation/admin.rst [hunk 1]
```diff
@@ -375,9 +375,7 @@ NodeSelector_ to the ``cilium-ds.yaml`` file like this:
 
 	spec:
 	  template:
-	    spec:
-	      nodeSelector:
-	        with-network-plugin: cilium
+            spec: nodeSelector: with-network-plugin: cilium
```
Comment (michi-covalent) [inline Documentation/admin.rst]: probably doesn't matter but it looks like it's indented by 3 spaces instead of 2.

File: Documentation/admin.rst [hunk 2]
```diff
@@ -387,6 +385,39 @@ And then label each node where Cilium should be deployed:
...
+In case pods were already running before the Cilium DaemonSet was deployed,
+these pods will still be connected using the previous networking plugin
+according to the CNI configuration. A typical example for this is the
+``kube-dns`` service which runs in the ``kube-system`` namespace by default.
+
+A simple way to change networking for such existing pods is to rely on the fact
+that Kubernetes automatically restarts pods in a Deployment if they are
+deleted, so we can simply delete the original kube-dns pod and the replacment
+pod started immediately after will have networking managed by Cilium.  In a
+production deployment deployment, this step could be performed as a rolling
```
Comment (michi-covalent) [inline Documentation/admin.rst]: production deployment deployment => production deployment

**Review by michi-covalent — APPROVED** [review]
looks good. a couple of minor comments.

**Comment by tgraf** [comment]
> looks good. a couple of minor comments.
> Show outdated

fixed

**Pull request merged**